docs/integrations/sources/close-com.md

```diff
@@ -109,6 +109,7 @@ The Close.com connector is subject to rate limits. For more information on this
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                |
 | :------ | :--------- | :------------------------------------------------------- | :----------------------------------------------------------------------------------------------------- |
+| 0.5.3 | 2024-07-01 | [40515](https://github.com/airbytehq/airbyte/pull/40515) | Add support for array type custom fields |
```

Comment:
```suggestion
| 0.5.4 | 2024-07-01 | [40515](https://github.com/airbytehq/airbyte/pull/40515) | Add support for array type custom fields |
```

@jtruty need you to fix this small typo. I don't have permission to edit your branch.

Reply:
done!